homework2/pizza_order.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Pizza Order</title>
+</head>
+
+<body>
+    <form method="post">
+        <h3>Pizza Order</h3>
+        <label for="name">Your name:</label>
+        <input type="text" id="name" name="name" minlength="3" maxlength="10" required>
+
+        <br><br>
+
+        <label for="emailInput">Your email:</label>
+        <input type="text" id="emailInput" name="email" required>
+
+        <br><br>
+
+        <label for="quantity">Quantity:</label>
+        <select name="quantity">
+            <option value="1">1</option>
+            <option value="1">2</option>
+            <option value="1">3</option>
+            <option value="1">4</option>
+            <option value="1">5</option>
+            <option value="1">6</option>
+            <option value="1">7</option>
+        </select>
+
+        <br><br>
+
+        <label>Topping:
+            <input type="radio" name="topping" value="supreme" checked>
+            None
+            <input type="radio" name="topping" value="supreme">
+            Supreme
+            <input type="radio" name="topping" value="vegetarian">
+            Vegetarian
+            <input type="radio" name="topping" value="hawaian">
+            Hawaian
+        </label>
```

Comment:
Mai corect era sa pui:

```suggestion
        <p>Topping:</p>
            <input type="radio" id="none" name="topping" value="supreme" checked>
            <label id="none">None</label>
            <input type="radio" id="supreme" name="topping" value="supreme">
            <label id="supreme">Supreme</label>
            <input type="radio" id="vegetarian" name="topping" value="vegetarian">
            <label id="vegetarian">Vegetarian</label>
            <input type="radio" id="hawaian" name="topping" value="hawaian">
            <label id="hawaian">Hawaian</label>
```
Astfel, fiecare input ii este atribuit un label.